client/app/components/tags-control/modal-template.html

```diff
@@ -3,7 +3,8 @@
   <h4 class="modal-title">Add/Edit Tags</h4>
 </div>
 <div class="modal-body">
-  <ui-select multiple append-to-body tagging tagging-label=" "
+  <!-- &#8203; is zero-width space - seems regular whitespace breaks components built-in search -->
+  <ui-select multiple append-to-body tagging tagging-label="&#8203;"
```

Comment:
Let's use "(Create new tag)" here instead.

Reply:
Done. Just added `(create new tag)` (with small first letter - IMHO it looks a bit better):
![image](https://user-images.githubusercontent.com/12139186/48700266-45c6bc80-ebf4-11e8-9187-6f54d5ce086c.png)